### PR TITLE
Add to Schedule modal unit tests

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Add unit tests for changes related to https://github.com/Rise-Vision/rise-vision-apps/pull/1832.

## Motivation and Context
Editor Improvements epic

## How Has This Been Tested?
Unit tests pass.
[stage-20]

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: web/scripts/template-editor/services/svc-template-editor-factory.js [hunk 1]
```diff
@@ -233,7 +233,8 @@ angular.module('risevision.template-editor.services')
         return scheduleSelectorFactory.checkAssignedToSchedules()
           .then(_publish)
           .catch(function() {
-            factory.isAssignedToSchedules = scheduleSelectorFactory.hasSelectedSchedules;    
+            factory.isAssignedToSchedules = scheduleSelectorFactory.hasSelectedSchedules;
+            return $q.reject();
```
Comment (alex-deaconu) [inline web/scripts/template-editor/services/svc-template-editor-factory.js:237]: @ezequielc to confirm, is this just for unit test purposes? Thanks.

File: web/partials/template-editor/components/component-schedules/component-schedules.html [hunk 2]
```diff
@@ -1,8 +0,0 @@
-<div class="attribute-editor-component">
```
Comment (alex-deaconu) [inline web/partials/template-editor/components/component-schedules/component-schedules.html:1]: @ezequielc assuming this partial being here was just a merge issue? Thanks.

File: web/scripts/template-editor/services/svc-template-editor-factory.js (hunk 1 above)
Comment (ezequielc) [reply web/scripts/template-editor/services/svc-template-editor-factory.js:237]: @alex-deaconu Yes, it was needed to fix some unit tests. No impact on the functionality. But I am revisiting this flag to fix the issue I mentioned previously, so I might change this in my next PR. Thanks!

File: web/partials/template-editor/components/component-schedules/component-schedules.html (hunk 2 above)
Comment (ezequielc) [reply web/partials/template-editor/components/component-schedules/component-schedules.html:1]: @alex-deaconu Yes, I think I duplicated it by mistake in my merge with mater. Thanks!

File: web/scripts/template-editor/services/svc-template-editor-factory.js (hunk 1 above)
Comment (alex-deaconu) [reply web/scripts/template-editor/services/svc-template-editor-factory.js:237]: @ezequielc sounds good, thanks!

**Pull request merged**